vars/testsInStage.groovy

```diff
@@ -32,7 +33,7 @@ boolean call(String tags) {
                                  exit 0
                              fi
                          else
-                             if ./launch.py --list ''' + tags + '''; then
+                             if ./launch.py --list ''' + verbose + tags + '''; then
```

Comment:
How/where does this exhibit when a stage that has no tests is skipped completely such that there are no steps at all to be viewed and see the output from this?  I suppose one has to know to open the bigger overall console log/pipeline.log artifact?

Reply:
I use the pipeline steps and search for `Get test list` to find the output.  Here's an example of the listing all the tags when no matches are found: https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13110/24/execution/node/696/log/

Reply:
Just double-checking before I land this that all supported (at least[1]) daos branches have support for this `--verbose` argument to `launch.py`.

[1] Obviously `master` and `release/2.4` are minimally required.  But what happens if somebody needs to do a test against 2.0 for whatever possible reason?  Could we assume that they will need to also backport any `launch.py --verbose` support needed since they will be using `pipeline-lib@master` when they do their testing?

Reply:
Calling launch.py with `--verbose` is supported in master and all numbered release/* branches (2.4 through 0.9).  By itself calling `launch.py --list --verbose` will behave no different than `launch.py --list`.  When the master branch with https://github.com/daos-stack/daos/commit/a7578357c9016874c4011048f9180bdf599f7c2f commit is used with `launch.py --list --verbose` we will get a listing of all the available test tags that can be checked to see if there was an issue with the tags being used or the test files themselves when a match is not found.  This master branch change is currently being cherry-picked to release/2.4 and there are no other cherry-pick plans.